packaging/swift/update-ice-swift-nightly.sh

```diff
@@ -24,6 +24,10 @@ cd "$root_dir/packaging/swift"
 git clone "https://x-access-token:${ICE_NIGHTLY_PUBLISH_TOKEN}@github.com/zeroc-ice/ice-swift-nightly.git" -b main
 cd ice-swift-nightly
 
+# Clean up the repository to ensure files removed from ice are not kept in ice-swift-nightly
+rm -rfv *
```

Comment:
should we also include `./.*`?

Reply:
It would also remove ".git" which we need. Maybe a more correct approach is to list them first with `git ls-tree`